app/service/update-appeal-service.ts

```diff
@@ -417,7 +418,8 @@ export default class UpdateAppealService {
       },
       ...caseData.legalRepresentativeDocuments && { legalRepresentativeDocuments: this.mapDocsWithMetadataToEvidenceArray(caseData.legalRepresentativeDocuments) },
       ...caseData.tribunalDocuments && { tribunalDocuments: this.mapDocsWithMetadataToEvidenceArray(caseData.tribunalDocuments) },
-      documentMap: [ ...this.documentMap ]
+      documentMap: [ ...this.documentMap ],
+      hearingCentre: hearingCentre
```

Comment:
Can you please make hearingCentre conditional? this way we always mapping a hearingCentre even if theres one.
if you use something like we do to legalRepresentativeDocs or tribunalDocs we only would map it if hearingCentre exists.